lib/install/management/appliance.go

```diff
@@ -1203,14 +1205,15 @@ func (d *Dispatcher) CheckServiceReady(ctx context.Context, conf *config.Virtual
 	return nil
 }
 
-// vchFolder returns the namespaced folder for the vch or an error.
-func VchFolder(op trace.Operation, sess *session.Session, conf *config.VirtualContainerHostConfigSpec) (*object.Folder, error) {
+// vchFolder returns the namespaced folder for the VCH or an error.
+func vchFolder(op trace.Operation, sess *session.Session, conf *config.VirtualContainerHostConfigSpec) (*object.Folder, error) {
```

Comment:
Thank you for un exporting this since it was no longer needed outside of where it lives. :)